<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickConstituentFromDBDialog</class>
 <widget class="QDialog" name="PickConstituentFromDBDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pick constituent from database</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ql_name">
       <property name="text">
        <string>Constituent name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qle_constituentName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="qtbv_constituents"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
